app/views/admin/shared/_nested_sidebar.html.erb

```diff
@@ -1,55 +1,45 @@
-<% menu_items.each do |group_name, group| %>
-  <li>
-    <% if group[:children].length == 1 %>
-      <% if group[:children][0][:visible] %>
-        <a class="crayons-link crayons-link--block <%= "crayons-link--current" if deduced_scope(request) == group[:children][0][:controller] %>"
-           href="/admin/<%= group[:children][0][:controller] %>"
-           aria-current="<%= "page" if deduced_controller(request) == group[:children][0][:controller] %>"
-           data-action="click->sidebar#expandDropdown">
-          <%= crayons_icon_tag(group[:svg], class: "dropdown-icon", aria_hidden: true) %>
+<nav>
+  <ul class="m-0">
+    <li>
+      <a class="c-link c-link--block c-link--icon-left" href="<%= admin_path %>" aria-current="<%= "page" if controller.controller_name == "overview" %>">
+        <%= crayons_icon_tag("stack-line", class: "c-link__icon") %>
+        Overview
+      </a>
+    </li>
```

Comment:
@Ridhwana it feels naughty to me that this link is the special snowflake being outside admin_menu.rb but I'm not sure how to add it there.. Thoughts?

Reply:
@ludwiczakpawel i have this halfway working (except for showing it as the current link when we land on the page), I'll fix it in a subsequent PR.